services/todo-legacy/controllers/todo-controller.ts

```diff
@@ -0,0 +1,35 @@
+import { api, Application, inject } from '@loopback/core';
+import { def } from './todo-controller.api';
+import { TodoRepository } from '../repositories/todo';
+import * as util from 'util';
+
+@api(def)
+export class TodoController {
+  @inject('repositories.todo')
+  repository: TodoRepository;
+
+  constructor() {
+  }
+
+  async getTodo(id?: number, filter?: any) {
```

Comment:
Not according to this [line](https://github.com/strongloop/loopback-next-example/pull/60/files#diff-530a87784f9335a4eb7239ca307dfd0aR12)